srv/components/system/teardown.sls

```diff
@@ -20,10 +20,6 @@ Remove cortx-py-utils dependencies:
     - requirements: salt://components/system/files/cortx_py_utils_requirements.txt
     - bin_env: /usr/bin/pip3
 
-Remove cortx-py-utils pip package:
```

Comment:
Why are we removing this from teardown? I think we are uninstalling all the RPMs for teardown right...

Reply:
we should not be removing foundation component rpm right?

Reply:
this should be done for pre-factory and it is already there in foundation mini provisioner call, this was redundant state which was written. so I have removed it.